src/app/qgsattributetabledialog.cpp

```diff
@@ -631,6 +631,35 @@ void QgsAttributeTableDialog::layerActionTriggered()
 
   QgsAction action = qAction->data().value<QgsAction>();
 
+  switch ( action.type() )
+  {
+    case Qgis::AttributeActionType::GenericPython:
+    case Qgis::AttributeActionType::Mac:
+    case Qgis::AttributeActionType::Windows:
+    case Qgis::AttributeActionType::Unix:
+    {
+      const bool allowed = QgsGui::pythonEmbeddedInProjectAllowed( QgsProject::instance() );
```

Comment:
Would be nice to rename this method to `allowExecutionOfEmbeddedScripts` or something else generic